src/utils/constants.ts

```diff
@@ -25,3 +25,8 @@ export const UnitMeasure = {
     MILLIMETERS: ' мм',
     KILOMETERS: ' км',
 };
+
+export const MainBottomSheet = {
```

Comment:
As soon as i saw this name, I wondered what exactly it meant. It contains titles, so rename to MainBottomSheetTitles